plugins/gcs/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.19...3.24 FATAL_ERROR)
+
+find_package(Arrow REQUIRED CONFIG)
+if (NOT ${ARROW_GCS} OR NOT ${ARROW_CS} STREQUAL "ON")
+  message(WARNING "Disabling gcs plugin because the ARROW_GCS flag is not found/enabled. You probably need to enable ARROW_GCS when installing Apache Arrow.")
+  return()
+endif()
+
+project(
+  gcs
```

Comment:
This needs to be enabled in CI (just search for `parquet` in `.github/workflows/tenzir.yaml`) and also in the Nix build (see `nix/tenzir/default.nix`).

Reply:
Added `gcs` to CI & Nix.

Reply:
The GCS CI step fails because it can't find Arrow: https://github.com/tenzir/tenzir/actions/runs/6172780703/job/16755081478?pr=3498#step:17:29

Reply:
I probably have to add an Arrow installation step to `workflows/tenzir.yaml`, right?

Reply:
So instead of finding the package - can we expose the `ARROW_GCS` variable from the `libtenzir` `find_package(ARROW)` result somehow?

Reply:
Let's figure out why it fails before we come up with a solution.

Reply:
We found out why it failed - using `find_package` before a `project` declaration makes CMake not search for CMake package in Linux, since CMake probably  can't deduct the system at that point.

Should be fixed now.